plasma_framework/test/src/exits/payment/PaymentStandardExitable.test.js

```diff
@@ -0,0 +1,182 @@
+const PaymentStandardExitable = artifacts.require('PaymentStandardExitable');
+const FakePlasmaFramework = artifacts.require('FakePlasmaFramework');
+const DummyVault = artifacts.require('DummyVault');
+const ExitId = artifacts.require('ExitIdWrapper');
+const IsDeposit = artifacts.require('IsDepositWrapper');
+const ExitableTimestamp = artifacts.require('ExitableTimestampWrapper');
+
+const { BN, constants, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { MerkleTree } = require('../../../helpers/merkle.js');
+const Testlang = require('../../../helpers/testlang.js');
+
+
+contract('PaymentStandardExitable', ([_, alice, bob]) => {
+    const STANDARD_EXIT_BOND = 31415926535; // wei
+    const ETH = constants.ZERO_ADDRESS;
+    const CHILD_BLOCK_INTERVAL = 1000;
+    const MIN_EXIT_PERIOD = 60 * 60 * 24 * 7; // 1 week
+
+    const buildTestData = (amount, blockNum) => {
+        const tx = Testlang.deposit(amount, alice, ETH);
+        const utxoPos = Testlang.buildUtxoPos(blockNum, 0, 0);
+        const merkleTree = new MerkleTree([tx], 3);
+        const merkleProof = merkleTree.getInclusionProof(tx);
+
+        return {
+            utxoPos, tx, merkleTree, merkleProof,
+        };
+    };
+
+    describe('startStandardExit', () => {
+        before(async () => {
+            this.exitIdHelper = await ExitId.new();
+            this.isDeposit = await IsDeposit.new(CHILD_BLOCK_INTERVAL);
+            this.exitableHelper = await ExitableTimestamp.new(MIN_EXIT_PERIOD);
+        });
+
+        beforeEach(async () => {
+            this.framework = await FakePlasmaFramework.new();
+            this.exitGame = await PaymentStandardExitable.new(this.framework.address);
+            this.vault = await DummyVault.new();
+        });
+
+        it('should fail when cannot prove the tx is included in the block', async () => {
+            const testAmount = 0;
```

Comment:
Let's  make testAmount non-zero. If someone changes order of requirements in the contract there will be no need to refactor this test.

Reply:
nice catch. It was a mistake during copy pasting haha